references/cli/addon.go

```diff
@@ -431,6 +431,9 @@ func waitApplicationRunning(obj *v1beta1.Application) error {
 		timeConsumed := int(time.Since(start).Seconds())
 		applySpinnerNewSuffix(spinner, fmt.Sprintf("Waiting addon application running. It is now in phase: %s (timeout %d/%d seconds)...",
 			phase, timeConsumed, int(timeout.Seconds())))
+		if timeConsumed > int(timeout.Seconds()) {
+			return errors.Errorf("Enable timeout, please run \"vela status %s -n vela-system\" to check addon application status", obj.Name)
```

Comment:
```suggestion
			return errors.Errorf("Enabling timeout, please run \"vela status %s -n vela-system\" to check the status of the addon", obj.Name)
```

We don't need to emphasize an add-on id deployed through an application, which is internal implementation details.